File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1181.cs
using System;

class URI
{
    static void Main() {
        int n = int.Parse(System.Console.ReadLine().Trim());
        string e = System.Console.ReadLine().Trim();
        double soma = 0, valor;
        int i, j;
        for(i=0;i<12;i++){
            for(j=0;j<12;j++){
                valor = double.Parse(System.Console.ReadLine().Trim());
                if(i==n){
                    soma+=valor;
                }
            }
        }
        
        if(e=="S"){
            Console.Write("{0:f1}\n", soma);
        }else{
            Console.Write("{0:f1}\n", soma/12);
        } 
    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1181.c
#include <stdio.h>
#include <string.h>

int main(){
    int n;
    char e[4];
    double soma = 0, valor;
    int i, j;
    
    scanf("%d %s", &n, e);
    for(i=0;i<12;i++){
        for(j=0;j<12;j++){
            scanf("%lf", &valor);
            if(i==n){
                soma+=valor;
            }
        }
    }
    
    if(strcmp(e, "S")==0){
        printf("%.1f\n", soma);
    }else{
        printf("%.1f\n", soma/12);
    } 

    return 0;
}